src/node_crypto.cc

```diff
@@ -3952,6 +3953,37 @@ void SignBase::CheckThrow(SignBase::Error error) {
   }
 }
 
+int SignBase::GetRSAOptions(Environment *env, v8::Local<v8::Object> options,
+                            int *padding, int *saltlen) {
+  MaybeLocal<Value> maybePadding = options->Get(env->padding_string());
+  if (maybePadding.IsEmpty()) return 0;
+
+  Local<Value> paddingValue = maybePadding.ToLocalChecked();
+  if (paddingValue->IsNumber()) {
+    switch (paddingValue->Int32Value()) {
+    case RSA_PKCS1_PADDING:
+    case RSA_PKCS1_PSS_PADDING:
+      *padding = paddingValue->Int32Value();
+      break;
+    default:
+      env->ThrowError("Padding must be RSA_PKCS1_PADDING or "
```

Comment:
nit: lowercase `padding` (or generally whatever it corresponds to in the documentation)